dspace/config/modules/authority.cfg

```diff
@@ -0,0 +1,15 @@
+#---------------------------------------------------------------#
+#----------------- AUTHORITY CONFIGURATIONS --------------------#
+#---------------------------------------------------------------#
+# These configs are used by the authority framework             #
+#---------------------------------------------------------------#
+
+#####  Authority Control Settings  #####
+
+plugin.named.org.dspace.content.authority.ChoiceAuthority = \
+org.dspace.content.authority.EPersonAuthority = EPersonAuthority
```

Comment:
While I'm not against creating a separate `authority.cfg` file... I wanted to note that we already have other `ChoiceAuthority` example settings in `dspace.cfg`.  

So, we may want to move the other `ChoiceAuthority` settings from `dspace.cfg` over to this file as well.  Either that, or move these settings to `dspace.cfg`.  It doesn't make sense to have some Authority settings in dspace.cfg while others are in authority.cfg.

Here's the authority control settings in dspace.cfg: https://github.com/DSpace/DSpace/blob/main/dspace/config/dspace.cfg#L1379-L1465

Reply:
I will move the already existing example in the new authority.cfg